libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -572,6 +607,7 @@ void AC_AutoTune_Heli::load_gain_set(AxisType s_axis, float rate_p, float rate_i
         attitude_control->get_rate_roll_pid().slew_limit(smax);
         attitude_control->get_angle_roll_p().kP(angle_p);
         attitude_control->set_accel_roll_max_cdss(max_accel);
+        attitude_control->set_ang_vel_roll_max_degs(max_rate);
```

Comment:
I don't think these are ever saved?

Reply:
These are only set to allow user to limit rates during autotune, if desired.  autotune does not determine max rates.

Reply:
Do we need a load call to put it back to the user's value after the tune is done in that case?